fairlearn/metrics/_plotter.py

```diff
@@ -0,0 +1,259 @@
+        if tup[0] > tup[1]:
+            raise ValueError(_CONF_INTERVALS_FLIPPED_BOUNDS_ERROR)
+
+
+def plot_metric_frame(metric_frame: MetricFrame,
+                      kind: str = "scatter",
+                      metrics: Union[List[str], str] = None,
+                      error_bars: Union[List[str], str] = None,
+                      conf_intervals: Union[List[str], str] = None,
+                      axs=None,
+                      show_plot=True,
+                      plot_error_labels: bool = False,
+                      text_precision_digits: int = 4,
+                      text_fontsize: int = 8,
+                      text_color: str = "black",
+                      text_ha: str = "center",
+                      capsize=10,
+                      colormap="Pastel1",
+                      subplot_shape=None,
+                      figsize=None
+                      ):
+    """Visualization for metrics with statistical error bounds.
+
+    Plots a given metric and its given error (as described by the `error_bars` or `conf_intervals`)
+
+    This function takes in a :class:`fairlearn.metrics.MetricFrame` with precomputed metrics
+    and metric errors and a `error_bars` or `conf_intervals` array to interpret the columns
+    of the `MetricFrame`.
+
+    The items at each index of the given `metrics` array and given `error_bars` or `conf_intervals`
+    array should correspond to a pair of the same metric and metric error, respectively.
+
+    Note:
+        If given many metrics to plot, this function plots them in only 1 row
+        with the number of subplot columns matching the number of metrics
+
+    Parameters
+    ----------
+    metric_frame : fairlearn.metrics.MetricFrame
+        The collection fo disaggregated metric values, along with the metric errors.
+
+    kind : str, optional
+        The type of plot to display. i.e. "bar", "line", etc.
+        List of options is detailed in `pandas.DataFrame.plot`
+
+    metrics : str or list of str
+        The name of the metrics to plot. Should match columns from the given `MetricFrame`.
+
+    error_bars : str or list of str
+        The name of the error metrics to plot. Should match columns from the given `MetricFrame`.
+        Error bars quantify the bounds relative to the base metric.
+
+        Example:
+            If the error bar for a certain column is [0.01, 0.02]
+            and the metric is 0.6
+            Then the plotted bounds will be [0.59, 0.62]
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper errors
+
+    conf_intervals : str or list of str
+        The name of the confidence intervals to plot.
+        Should match columns from the given `MetricFrame`.
+
+        Note:
+            The return of the error bar function should be a tuple of the lower
+            and upper bounds
+
+    axs : matplotlib.axes._axes.Axes or list of matplotlib.axes._axes.Axes, optional
```

Comment:
It should be possible to turn these MatplotLib things into links too - the relevant files are pulled in by `docs/conf.py`